ocpmodels/models/pretrained_models.yml

```diff
@@ -0,0 +1,67 @@
+"CGCNN 200k": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2020_11/s2ef/cgcnn_200k.pt"
+"CGCNN 2M": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2020_11/s2ef/cgcnn_2M.pt"
+"CGCNN 20M": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2020_11/s2ef/cgcnn_20M.pt"
+"CGCNN All": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2020_11/s2ef/cgcnn_all.pt"
+"DimeNet 200k": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2020_11/s2ef/dimenet_200k.pt"
+"DimeNet 2M": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2020_11/s2ef/dimenet_2M.pt"
+"SchNet 200k": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2020_11/s2ef/schnet_200k.pt"
+"SchNet 2M": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2020_11/s2ef/schnet_2M.pt"
+"SchNet 20M": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2020_11/s2ef/schnet_20M.pt"
+"SchNet All": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2020_11/s2ef/schnet_all_large.pt"
+"DimeNet++ 200k": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2021_02/s2ef/dimenetpp_200k.pt"
+"DimeNet++ 2M": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2021_02/s2ef/dimenetpp_2M.pt"
+"DimeNet++ 20M": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2021_02/s2ef/dimenetpp_20M.pt"
+"DimeNet++ All": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2021_02/s2ef/dimenetpp_all.pt"
+"SpinConv 2M": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2021_12/s2ef/spinconv_force_centric_2M.pt"
+"SpinConv All": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2021_08/s2ef/spinconv_force_centric_all.pt"
+"GemNet-dT 2M": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2021_12/s2ef/gemnet_t_direct_h512_2M.pt"
+"GemNet-dT All": "https://dl.fbaipublicfiles.com/opencatalystproject/models/2021_08/s2ef/gemnet_t_direct_h512_all.pt"
+"PaiNN All": "https://github.com/Open-Catalyst-Project/ocp/blob/main/configs/s2ef/all/painn/painn_nb6_scaling_factors.pt"
```

Comment:
Thanks for this PR! I was just randomly reading through this PR and found that PaiNN and GemNet have scaling factor paths instead of checkpoints. I don't think that's intended (?)

Reply:
@AdeeshKolluru Thank you for randomly reading through! I fixed the links, maybe its worth writing a test to load all these

Reply:
I think you could just change the current model tests to load paths from  `pretrained_models.yml` instead of the paths that are manually entered. For ex - [here](https://github.com/Open-Catalyst-Project/ocp/blob/main/tests/models/test_gemnet_oc.py#L50)

Reply:
I agree with @AdeeshKolluru suggestion!

For context @misko, I put the model URLS [here](https://github.com/Open-Catalyst-Project/ocp/blob/869f1341f03f8a1bf6bfab1fe0a7e1bf6a916daf/ocpmodels/models/pretrained_models.yml)